project-base/storefront/components/Blocks/Login/LoginForm.tsx

```diff
@@ -60,6 +63,19 @@ export const LoginForm: FC<LoginFormProps> = ({
         return url;
     };
 
+    const getSocialNetworkIcon = (socialNetwork: TypeLoginTypeEnum) => {
+        switch (socialNetwork) {
+            case TypeLoginTypeEnum.Facebook:
+                return <FacebookIcon className="w-7 text-[#1877f2]" />;
+            case TypeLoginTypeEnum.Google:
+                return <GoogleIcon className="w-6" />;
+            case TypeLoginTypeEnum.Seznam:
+                return <SeznamIcon className="w-6" />;
+            default:
+                return null;
+        }
+    };
+
```

Comment:
I would convert it to a stateless functional component instead of using it as a getter function. The intent of the component would be clearer and we could use the `<SocialNetworkIcon ... />` syntax.

```suggestion
    const SocialNetworkIcon = ({ socialNetwork }: { socialNetwork: TypeLoginTypeEnum }) => {
        switch (socialNetwork) {
            case TypeLoginTypeEnum.Facebook:
                return <FacebookIcon className="w-7 text-[#1877f2]" />;
            case TypeLoginTypeEnum.Google:
                return <GoogleIcon className="w-6" />;
            case TypeLoginTypeEnum.Seznam:
                return <SeznamIcon className="w-6" />;
            default:
                return null;
        }
    };

```

Reply:
And I would also move it outside of the scope of the `LoginForm`.

Reply:
:ok_hand: fixed in 98beaf69d1